### PR TITLE
ODF 4.15 Release

### PR DESCRIPTION
**Pull request by aayushsss1**
ODF 4.15 Release 

1. Added three new parameters - 

- resourceProfile
- enableNFS
- useCephRBDAsDefaultStorageClass

2. Updated the Deployment File

### CODE REVIEW

File: config-templates/redhat/odf-local/changelog.json [hunk 1]
```diff
@@ -1,4 +1,21 @@
 [
+    {
+        "4.15": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "6 May 2024",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: Update release date to `09 May 2024`

File: config-templates/redhat/odf-remote/changelog.json [hunk 2]
```diff
@@ -1,4 +1,21 @@
 [
+    {
+        "4.15": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "6 May 2024",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: same here

**Review by mssachan — CHANGES_REQUESTED** [review]
Added minor comment. 
Also get approval from @derekpoindexter for the description of new parameters.

File: config-templates/redhat/odf-local/4.15/custom-parameters.json [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+    "regex": "^(?:true|false)$",
+    "min-length": 4,
+    "max-length": 5,
+    "supported-values": ["true", "false"],
+    "type": "boolean",
+    "required": "true",
+    "mutable": "false"
+  },
+  {
+    "description": "Enable to set the Ceph RADOS block device (RBD) storage class as the default storage class during the deployment of OpenShift Data Foundation",
```
Comment (derekpoindexter) [inline config-templates/redhat/odf-local/4.15/custom-parameters.json]:
```suggestion
    "description": "Set the Ceph RADOS block device (RBD) storage class as the default storage class during the deployment of OpenShift Data Foundation",
```

File: config-templates/redhat/odf-local/4.15/custom-parameters.json [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+    "regex": "^(?:true|false)$",
+    "min-length": 4,
+    "max-length": 5,
+    "supported-values": ["true", "false"],
+    "type": "boolean",
+    "required": "true",
+    "mutable": "false"
+  },
+  {
+    "description": "Provides an option to choose a resource profile based on the availability of resources during deployment. Choose between 'lean', 'balanced' and 'performance'.",
```
Comment (derekpoindexter) [inline config-templates/redhat/odf-local/4.15/custom-parameters.json]:
```suggestion
    "description": "Choose a resource profile based on the availability of resources during deployment. Choose between 'lean', 'balanced' and 'performance'.",
```

File: config-templates/redhat/odf-local/4.15/custom-parameters.json [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+    "regex": "^(?:true|false)$",
+    "min-length": 4,
+    "max-length": 5,
+    "supported-values": ["true", "false"],
+    "type": "boolean",
+    "required": "true",
+    "mutable": "false"
+  },
+  {
+    "description": "Enabling this allows you to create exports using Network File System (NFS) that can then be accessed internally or externally from the OpenShift cluster",
```
Comment (derekpoindexter) [inline config-templates/redhat/odf-local/4.15/custom-parameters.json]:
```suggestion
    "description": "Allow exports from the Network File System (NFS) that can then be accessed internally or externally from the OpenShift cluster",
```

**Review by derekpoindexter — CHANGES_REQUESTED** [review]
see suggestions

**Review by derekpoindexter — APPROVED** [review]

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**